packages/app/src/app/overmind/effects/vscode/Workbench.ts

```diff
@@ -361,7 +361,7 @@ export class Workbench {
     addBrowserNavigationCommand(
       'codesandbox.help.discord',
       'Join our discord server',
-      'https://discord.gg/5BpufEP7MH'
+      'https://discord.gg/C6vfhW3H6e'
```

Comment:
Hey, any reason why the link has changed? I guess I also need to update on Sandpack documentation

Reply:
Hi @danilowoz, the link was changed so that we can look at this particular invite link from Discord and determine how many people are coming in from the client dashboard.